modules/hub/src/OnchainTransactionService.ts

```diff
@@ -333,6 +366,53 @@ export class OnchainTransactionService {
       return
     }
 
+    if (txn.state == 'pending_failure') {
+      if (!txn.hash) {
+        await this.signAndSendTx(txn)
```

Comment:
The send should be unconditional, but happen after the poll to see if it exists (refer to the flowchart)